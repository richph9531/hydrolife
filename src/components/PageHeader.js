import React from 'react';

import classes from '../styles/PageHeader.module.css';

const PageHeader = () => (
  <div className={classes.PageHeader}>
    <div className={classes.LogoContainer}>
      <img 
        src={`${process.env.PUBLIC_URL}/media/logo2.png`}
        alt='Hydrolife Logo'/>
      </div>
    <div className={classes.MenuContainer}>
      <img
        src={`${process.env.PUBLIC_URL}/media/menu2.png`}
        alt='Menu'/>
    </div>
  </div>
);

export default PageHeader;
