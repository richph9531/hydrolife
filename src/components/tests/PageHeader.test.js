import React from 'react';

import { render } from '@testing-library/react';
import PageHeader from '../PageHeader';

import contactUsData from '../../data/contactUs.json'

test('Page header renders correctly', () => {
  const view = render(<PageHeader data={contactUsData}/>);
  expect(view.baseElement).toMatchSnapshot();
});
