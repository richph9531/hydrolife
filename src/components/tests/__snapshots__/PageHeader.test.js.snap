// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page header renders correctly 1`] = `
<body>
  <div>
    <div
      class="PageHeader"
    >
      <div
        class="LogoContainer"
      >
        <img
          alt="Hydrolife Logo"
          src="/media/logo.png"
        />
      </div>
      <div
        class="ContactUsContainer"
      >
        <div>
          <div
            class="ContactIconsForHeader"
          >
            <a
              href="tel:0878203953"
              rel="_blank"
              target="noopener noreferrer"
            >
              <img
                alt="tel:0878203953"
                src="/media/landline.svg"
              />
            </a>
          </div>
          <div
            class="ContactIconsForHeader"
          >
            <a
              href="https://wa.me/27844297382"
              rel="_blank"
              target="noopener noreferrer"
            >
              <img
                alt="https://wa.me/27844297382"
                src="/media/whatsapp.svg"
              />
            </a>
          </div>
          <div
            class="ContactIconsForHeader"
          >
            <a
              href="https://www.instagram.com/p/Ciz_zXgqCCn/"
              rel="_blank"
              target="noopener noreferrer"
            >
              <img
                alt="https://www.instagram.com/p/Ciz_zXgqCCn/"
                src="/media/instagram.svg"
              />
            </a>
          </div>
          <div
            class="ContactIconsForHeader"
          >
            <a
              href="https://www.facebook.com/profile.php?id=100079179708997"
              rel="_blank"
              target="noopener noreferrer"
            >
              <img
                alt="https://www.facebook.com/profile.php?id=100079179708997"
                src="/media/facebook.svg"
              />
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
